ietf/doc/templatetags/ietf_filters.py

```diff
@@ -142,45 +125,11 @@ def bracketpos(pos,posslug):
 
 register.filter('fill', fill)
 
-@register.filter(name='rfcspace')
-def rfcspace(string):
-    """
-    If the string is an RFC designation, and doesn't have
-    a space between 'RFC' and the rfc-number, a space is
-    added
-    """
-    string = str(string)
-    if string[:3].lower() == "rfc" and string[3] != " ":
-        return string[:3].upper() + " " + string[3:]
-    else:
-        return string
-
-@register.filter(name='rfcnospace')
-def rfcnospace(string):
```

Comment:
The ability to have more than one representation (with and without space) was intentional. While we moved most of the datatracker to having the space, there was concern about disrupting the workflow of external entities using the IPR system. If changing the output wasn't what you meant to do, there's some further suggestion below.